### DISCUSSION TITLE
is LICENSE going to change into MIT?

### DISCUSSION THREAD

**Discussion by skimhugo** [spacedriveapp/spacedrive#182] [General]
is LICENSE going to change into MIT?

**Comment by oscartbeaumont** [comment]
~I can't see why it would~. Please refer to the [LICENCE](https://github.com/spacedriveapp/spacedrive/blob/main/LICENSE) for the current license.

I can see the `package.json`'s have the LICENCE set to `MIT`. This is incorrect and will be fixed shortly.
